src/styles.css

```diff
@@ -1,6 +1,10 @@
 /* Make clicks pass-through */
 #nprogress {
   pointer-events: none;
+
+  /* Ensures progress container does not interfere with layout */
+  position: absolute;
+  width: 100%;
```

Comment:
We actually could consider removing this and letting the consumer decide how to position it.

Reply:
I think I've talked myself out of it. I'm going to remove the styles.